rma/models/rma.py

```diff
@@ -584,6 +584,7 @@ def action_refund(self):
             invoice_form = Form(self.env['account.invoice'].with_context(
                 default_type='out_refund',
                 company_id=rmas[0].company_id.id,
+                tracking_disable=True,
```

Comment:
```suggestion
            invoice_form = Form(self.env['account.invoice'].sudo().with_context(
                default_type='out_refund',
                company_id=rmas[0].company_id.id,
```

Reply:
IMO it's better solution.

Reply:
Also, if I try to reproduce the error in the following way:
- Install `rma`
- Install `mail_tracking`
- Configure demo user with: RMA permissions, Account permissions and not Settings permission
- Create RMA
- Refund RMA

It doesn't show me the error.
Do I need any other addon installed and / or configured?

Reply:
mail_tracking is not needed. The table is in core. sudo is not valid, as it changes other values like company.

Reply:
Ok, `mail_tracking` is not required but i can't reproduce.

Reply:
Are you testing with demo user?

Reply:
Yes, demo user in local